general/app/development/links/links.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Links in the Moodle App
+sidebar_label: Links
```

Comment:
```suggestion
title: Inner links in the Moodle App
sidebar_label: Inner links
```

I'm not 100% convinced about this suggestion, but I'd like to differentiate this concept from "deep links". Is it inner links vs deep links? Or how else could we refer to them?

Reply:
As we talked in chat, I've used "Links in the app" and "Deep linking" as section titles.